python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -19,9 +21,9 @@
 from ray.rllib.evaluation.policy_graph import PolicyGraph
 from ray.rllib.evaluation.tf_policy_graph import TFPolicyGraph
 
-ACTION_SCOPE = "a_func"
-POLICY_SCOPE = "p_func"
-POLICY_TARGET_SCOPE = "target_p_func"
+ACTION_SCOPE = "action"
+POLICY_SCOPE = "policy"
+POLICY_TARGET_SCOPE = "target_policy"
 Q_SCOPE = "q_func"
```

Comment:
If we have changed to fullname style, it would be more unified to use 'critic/value' rather than 'q_func'